share/spice/sports/mlb/games/sports_mlb_games.js

```diff
@@ -0,0 +1,175 @@
+(function (env) {
+    "use strict";
+
+    env.ddg_spice_sports_mlb_games = function(apiResult) {
+
+        if (!apiResult || !apiResult.data || !apiResult.data.games || !apiResult.data.games.length) {
+            return Spice.failed('mlb_games');
+        }
+        
+        DDG.require('moment.js', function(){
+            Spice.add({
+                id: 'mlb_games',
+                name: 'MLB Games',
+
+                data: gameData(apiResult.data),
+
+                from: apiResult.from,
+                signal: apiResult.signal,
+
+                meta: {
+                    idField: 'id',
+                    showMoreAt: true,
+                    sourceIcon: false,
+                    sourceUrl: "http://www.bleacherreport.com/",
+                    sourceName: 'Bleacher Report',
+                    secondaryText: '<span class="tx-clr--grey-dark">Data from SportsData</span>',
+                    hideModeSwitch: true,
```

Comment:
Cool, just double checking that we intend to start hiding that thing going forward. We had initially hid it in bttf and then added it back, disabled.